pandas/tests/plotting/frame/test_frame.py

```diff
@@ -1520,11 +1532,11 @@ def test_errorbar_plot(self):
             ax = _check_plot_works(s_df.plot, y="y", x="x", yerr=yerr)
             self._check_has_errorbars(ax, xerr=0, yerr=1)
 
-        with pytest.raises(ValueError):
+        with tm.external_error_raised(ValueError):
             df.plot(yerr=np.random.randn(11))
 
         df_err = DataFrame({"x": ["zzz"] * 12, "y": ["zzz"] * 12})
-        with pytest.raises((ValueError, TypeError)):
+        with tm.external_error_raised(TypeError):
```

Comment:
not sure why this previously allowed it to be a `ValueError` as well as a `TypeError`